### PR TITLE
Add css cursor

### PR DESCRIPTION
**Pull request by funkhouserw**
adds css cursor attribute to helper so we can see how clickable some elements are.

### CODE REVIEW

**Comment by funkhouserw** [comment]
This appears to work on the heroku PR build. 

Clicking a clickable element/button results in element.cursor  coming in as 'pointer', where as other clicks return 'auto' or 'text'. 

Note that this won't always return what we want, there are plenty of exceptions.  I'm OK with that:  if it can help me 20% it's worth it.

File: lib/helpers/getDomNodeProfile.js [hunk 1]
```diff
@@ -26,7 +26,8 @@ export function getDomNodeProfile(el, options = {}) {
     selector: getDomNodePath(el),
     tag_name: el.tagName,
     text: el.text,
-    text_content: getTextContent(el, options),
+    cursor:  document.defaultView.getComputedStyle(el).getPropertyValue('cursor'),
```
Comment (chanson) [inline lib/helpers/getDomNodeProfile.js]: nit: unless there's a specific reason for it to be here, can we keep the list alphabetized?

Comment (chanson) [reply lib/helpers/getDomNodeProfile.js]: This is probably fine, but I don't think `getComputedStyle` is the most performant method. In the future, maybe we can come up with some way to track how long it's taking events to send to make sure we're not inadvertently creating bottlenecks. If we had short term concerns we could do something like:
```
let cursor = el.style.cursor;
if ( ! cursor ) {
  cursor = window.getComputedStyle(el).getPropertyValue('cursor');
}​
```

Comment (chanson) [reply lib/helpers/getDomNodeProfile.js]: Also fwiw, it sounds like you no longer need to do `.getPropertyValue('cursor')` you can just call `.cursor`, and it also sounds like you can call `getComputedStyle` on `window` - not sure if that has performance implications, so do what you will with that knowledge

**Comment by chanson** [comment]
Needs a rebase and `package.json` version bump

**Pull request merged**